source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -1,6 +1,7 @@
 #include "extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.h"
 
 #include <memory>
+#include <set>
```

Comment:
Why not unordered_set?

Reply:
OK that's fine. We might want to change the upstream project to use `unordered_set` as I see no reason not to do that but not a big deal for now. cc @lizan

Reply:
Sounds good. I will submit a PR to the upstream project. :)